integrations/docker/images/chip-build-asr/Dockerfile

```diff
@@ -1,20 +1,24 @@
 ARG VERSION=latest
 FROM connectedhomeip/chip-build:${VERSION}
 
-# Setup ASR
+RUN set -x \
+    && apt-get update \
+    && DEBIAN_FRONTEND=noninteractive apt-get install -fy --no-install-recommends \
+    wget \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/ \
+    && : # last line
+
+# Setup toolchain for ASR582X
 ARG ASR_DIR=/opt/asr
```

Comment:
Maybe export this directory as `ASR_SDK_ROOT`. Many other platforms do that, e.g.: `ENV BOUFFALOLAB_SDK_ROOT=/opt/bouffalolab_sdk`